debian/landscape-client.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Landscape client daemons
+Documentation=man:landscape-client(1)
+Documentation=man:landscape-config(1)
+After=local-fs.target
+After=remote-fs.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Type=forking
+Group=landscape
+ExecStart=/usr/bin/landscape-client --daemon --pid-file /run/landscape/landscape-client.pid
```

Comment:
hmm, won't we be better off not forking (i.e. no `--daemon`)and having `systemd` invoke `landscape-client` directly?

Reply:
It doesn't make a lot of difference here. Everything is ultimately tracked with cgroups.

I'd have to test a bit more if we change to a simple service to make sure everything runs nice from all the entry points as I have some doubts.

Reply:
correction. I've changed to simple service as suggested. It detects better if the service fails to start for whatever reason.